packages/block-library/src/navigation/edit/index.js

```diff
@@ -280,9 +298,10 @@ function Navigation( {
 		! isCreatingNavigationMenu &&
 		! isConvertingClassicMenu &&
 		hasResolvedNavigationMenus &&
+		classicMenus?.length === 0 &&
 		! hasUncontrolledInnerBlocks;
 
-	if ( isPlaceholder && ! ref ) {
+	if ( isPlaceholder ) {
```

Comment:
We should probably explain why this was removed.

Reply:
we didn't remove it, it's already being checked as part of `isPlaceholder`.

Reply:
👍